pypi_count/command_line_interface.py

```diff
@@ -25,9 +25,12 @@ def main(  # pylint: disable=R0913
     function_with_docstrings: bool = typer.Option(False, "--function_with_docstrings"),
     class_with_docstrings: bool = typer.Option(False, "--class_with_docstrings"),
     class_without_docstrings: bool = typer.Option(False, "--class_without_docstrings"),
+    find_parameters: bool = typer.Option(False, "--find_parameters"),
```

Comment:
The `--comment`, `--function_def`, and `--class_def` options should be made plural to match the other countable options.

Reply:
@solisa986